src/api_v2/datasources.md

```diff
@@ -363,7 +363,7 @@ curl -H "X-API-Token: yourToken -X PUT http://rudder.example.com/rudder/api/late
 
   /**
   @api {post} /api/datasources/reload 6. Update properties from data source - all nodes
-  @apiVersion 4.0.0
+  @apiVersion 9.0.0
```

Comment:
it is not in the version 8?

Reply:
not, it was an error previously

Reply:
(datasources didn't existed at all before that, and actually, it's a plugin, so I'm not even sure it should be their, or we miss a lot of other plugins, but that discussion belongs elsewhere)